starter-bots/purely-functional-scala/project/plugins.sbt

```diff
@@ -0,0 +1,4 @@
+addCompilerPlugin(
+  "org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full
+)
```

Comment:
I don't think macro paradise is necessary for this bot, unless you are running on older versions of scala. Is there a particular reason you've added it?

Reply:
You're correct.  I misread the documentation for circe.
Will be removed on next push.